src/utils.test.ts

```diff
@@ -39,6 +42,34 @@ describe("convertSkylinkToBase32", () => {
   });
 });
 
+describe("getRelativeFilePath", () => {
+  // TODO: Is this right?
+  const filepaths = [
+    ["abc/def", "abc:def"],
+    ["./abc/def", ".:abc:def"],
+    ["/abc/def.txt", ":abc:def.txt"],
+  ];
+
+  it.each(filepaths)("the relative file path for a file with the path %s should be %s", (filepath, directory) => {
+    const file = new File(["test contents"], filepath);
+    expect(getRelativeFilePath(file)).toEqual(directory);
+  });
+});
+
+describe("getRootDirectory", () => {
+  // TODO: Is this right?
+  const filepaths = [
+    ["abc/def", "."],
+    ["./abc/def", "."],
+    ["/abc/def", "."],
```

Comment:
This looks wrong. I don't understand why it passes.

Reply:
I have the same feeling. I think these functions do not work properly in node. I would like to fix this in a followup with Karol's help.

Reply:
responded below

Reply:
@kwypchlo I see two mentions of "responded below" but I can't find the response itself. Could you please add a link to it?